### DISCUSSION TITLE
挂载本地硬盘，根文件夹路径只能是/home这一级的吗

### DISCUSSION THREAD

**Discussion by cloud-wyf** [AlistGo/alist#5719] [Q&A]
挂载本地硬盘，根文件夹路径填/home这一级就没问题，填/home/wy就报错failed get objs: failed to list objs: open /home/wy: no such file or directory

**Comment by HXTHETK** [comment]
你不看看他提示什么吗？他告诉你没有这样的目录，你要先建立这个文件夹

**Reply by cloud-wyf** [reply]
我肯定是有这个目录才问的啊，现在找到的可能是问题的解释是:docker镜像只能访问容器内的目录，但不清楚为什么能找到home，找不到/home/wy

**Reply by HXTHETK** [reply]
如果你没有用docker的-v参数挂载路径的话，容器和宿主机目录不共享，你可以尝试先挂载home，然后用网盘进入后新建wy，在尝试挂载，我这里2级目录3级目录的挂载都是正常的

**Comment by cloud-wyf** [comment]
而且在挂载/home成功的情况下，手动向/home文件夹添加东西，在alist也看不到

**Reply by HXTHETK** [reply]
你先别挂载home，直接挂载/目录，看看是什么情况，docker一般来说不会出问题的

**Reply by HXTHETK** [reply]
百度failed get objs是你没有权限操作目录

**Comment by walker2top** [comment]
你是不是用docker compose起的容器，我发现我用docker compose起容器就会挂载不上（容器起来了，inspect也能看到bind mount关系，但文件夹就是空的），我用docker run就没问题....
